VultisigApp/VultisigApp/Views/Vault/BackupPasswordSetupView.swift

```diff
@@ -57,23 +63,6 @@ struct BackupPasswordSetupView: View {
 #if os(macOS)
         .padding(.horizontal, 25)
 #endif
-        .fileExporter(
```

Comment:
once this get removed , on IOS , the file is named always "file.bak" , you need to update the one in `exportFile`